crates/ruff_linter/src/rules/pydoclint/rules/check_docstring.rs

```diff
@@ -407,20 +557,25 @@ struct DocstringSections<'a> {
     returns: Option<GenericSection>,
     yields: Option<GenericSection>,
     raises: Option<RaisesSection<'a>>,
+    parameters: Option<ParametersSection<'a>>,
 }
 
 impl<'a> DocstringSections<'a> {
     fn from_sections(sections: &'a SectionContexts, style: Option<SectionStyle>) -> Self {
         let mut docstring_sections = Self::default();
         for section in sections {
             match section.kind() {
+                SectionKind::Args | SectionKind::Arguments | SectionKind::Parameters => {
+                    docstring_sections.parameters =
+                        Some(ParametersSection::from_section(&section, style));
```

Comment:
Is it intentional that `SectionKind::` uses `Args`/`Arguments` and `ParametersSection` and `docstring_sections` use the term `Parameters`?

Reply:
The SectionKind is based on what the sections are usually called by convention. Google typically uses "Args" while NumPy typically uses "Parameters". I think the more correct term is parameters, so I called it `ParametersSection`.